mars/dataframe/expressions/tests/test_datasource.py

```diff
@@ -239,3 +241,12 @@ def testFromPandasSeries(self):
         self.assertTrue(series.chunks[2].index_value._index_value._is_monotonic_increasing)
         self.assertFalse(series.chunks[2].index_value._index_value._is_monotonic_decreasing)
         self.assertTrue(series.chunks[2].index_value._index_value._is_unique)
+
+    def testFromTensor(self):
+        tensor = mt.random.rand(10, 10, chunk_size=2)
+        df = from_tensor_mt(tensor)
+        self.assertIsInstance(df.index_value._index_value, IndexValue.RangeIndex)
+        self.assertEqual(df.op.dtypes, tensor.dtype, 'DataFrame converted from tensor have the wrong dtype')
+
+        df.tiles()
+        self.assertEqual(len(df.chunks), 25)
```

Comment:
Check more for chunks.

Reply:
OK. I check the index/column value for chunks.